### PR TITLE
about/welcome copy updates

### LINKED ISSUE #319: About page improvements
1. Bold *Collections* in the Welcome message modal
2. Capitalize "Scout" in the About page
3. Make About page less NYC-specific

### PR DESCRIPTION
**Pull request by indraneel**
About page and Welcome modal changes to close #319

### CODE REVIEW

File: packages/frontend/src/layout/AboutPage/AboutPage.jsx [hunk 1]
```diff
@@ -16,24 +16,24 @@ export default function AboutPage() {
     <div className="about-page">
       <section className="about-dark-header">
         <div className="about-header-full">
-          <h3>Introducing scout</h3>
+          <h3>Introducing Scout</h3>
```
Comment (jps327) [inline packages/frontend/src/layout/AboutPage/AboutPage.jsx:19]: Can we change this to "Welcome to Scout" or something else? Scout has been introduced a long time ago so this feels outdated

File: packages/frontend/src/components/WelcomeModal/WelcomeModal.jsx [hunk 2]
```diff
@@ -21,8 +21,9 @@ export default function WelcomeModal() {
           </p>
 
           <p>
-            You can also create collections of datasets which you can share with
-            others. To do so start collecting or click on the collections tab
+            You can also create <b>collections</b> of datasets which you can
+            share with others. To do so start collecting or click on the{' '}
```
Comment (jps327) [inline packages/frontend/src/components/WelcomeModal/WelcomeModal.jsx:25]: Add a comma after "To do so"
```
To do so, start...
```

File: packages/frontend/src/components/WelcomeModal/WelcomeModal.jsx [hunk 3]
```diff
@@ -21,8 +21,9 @@ export default function WelcomeModal() {
...
+            <b>Collections tab</b>.
```
Comment (jps327) [inline packages/frontend/src/components/WelcomeModal/WelcomeModal.jsx:26]: Only Collections should be bolded here `<b>Collections</b> tab`

**Review by jps327 — APPROVED** [review]
Looks good! Just some minor edits then go ahead and merge

**Review by indraneel — COMMENTED** [review]
copy updates

**Pull request merged**